modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3441,6 +3441,144 @@ Response::
 | String ID of the query.
 |===
 
+
+=== Get information about a template query
+
+`GET /gsql/library?functionName=pkg1.pkg2.<template name>`
+
+This endpoint returns the query definition, query name, and query parameters for a template query.
+
+==== Sample request
+[tabs]
+====
+Request::
++
+--
+[source.wrap,bash]
+----
+curl -X GET --user tigergraph:tigergraph --url 'http://localhost:14240/gsqlserver/library?functionName=gds.community.louvain'
```

Comment:
should be http://localhost:14240/gsqlserver/gsql/library?functionName=gds.community.louvain

Reply:
for the host `localhost`, you should make them consitent with current documentation style.